docs/README.md

```diff
@@ -15,27 +12,32 @@ If you are a newcomer and would like to learn more about the Cosmos SDK, this **
 
 If you like to learn by doing, you can follow the **[SDK application tutorial](https://github.com/cosmos/sdk-application-tutorial)**. It showcases how to build an SDK-based blockchain from scratch, and teaches you about the basic principles the SDK in the process.
 
-## Use the SDK
-
-The following sections contain the information you need if you want to build a fully-functional SDK-based blockchain:
+### Resources
 
->*NOTE*: We are currently working on improving the docs. Some info might be missing. If that is the case, try the Cosmos [Forum](https://forum.cosmos.network). Failing that, [open an issue](https://github.com/cosmos/cosmos-sdk/issues/new).
-
-- [Introduction](./intro/README.md): Contains introductory high-level material on the Cosmos SDK.
-- [Gaia](./gaia/README.md): Contains all documentation related to the gaia application (current name for the Cosmos-Hub). Also contains info on how to join gaia testnets.
-- [Clients](./clients/README.md): Documentation about SDK clients like the SDK Command-Line interface and the SDK Light-client.
-- [Specifications](./spec/README.md): Contains SDK and modules specifications.
-- [SDK API Reference](https://godoc.org/github.com/cosmos/cosmos-sdk): Godocs of the Cosmos-SDK.
+- [Specifications](./spec/README.md): Specifications of modules and other parts of the Cosmos SDK. 
+- [SDK API Reference](https://godoc.org/github.com/cosmos/cosmos-sdk): Godocs of the Cosmos SDK.
```

Comment:
Maybe we also add the CLI and testnet documentation here too?

Reply:
The CLI doc is a bit empty rn, and testnet docs linked just below